v3/src/components/data-display/data-display-types.ts

```diff
@@ -7,7 +7,7 @@ export type Point = { x: number, y: number }
 export type CPLine = { slope: number, intercept: number, pivot1?: Point, pivot2?: Point }
 export const kNullPoint = {x: -999, y: -999}
 
-export const PointDisplayTypes = ["points", "bars", "bins"] as const
+export const PointDisplayTypes = ["points", "bars", "bins", "histogram"] as const
```

Comment:
One thing I'm especially looking for thoughts on is the choice to add `histogram` as an option here. Does that seem reasonable? It seemed to provide the most straightforward way to check when the histogram display is active. In theory, you could instead use a combination of the `bins` option and the separate `graphModel.pointsFusedIntoBars`, but I found that more difficult to work with.

Reply:
I agree that it could be helpful to add "histogram" as a PointDisplayType